csi-plug-n-play/sidecars/soda-proxy/Readme.md

```diff
@@ -0,0 +1,37 @@
+## soda-proxy
+
+soda-proxy is a simple http server which helps to call the api's of SodaFoundation in a simpler way, it bypasses the authentication and directly helps to connect the osdsapi-server using the client provided by sodafoundation/api.
+
+### Build
+Please follow the below steps to build and run the proxy
+
+```go
+go get github.com/sodafoundation/nbp
+
+git checkout csipnp_dev 
```

Comment:
Once we merge, this step will be wrong? Can we add a comment

Reply:
@skdwriting I have removed this step, as we will raise the PR to master today itself.